PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditResult.kt

```diff
@@ -1,42 +1,40 @@
 package com.braintreepayments.api.paypal.vaultedit
 
-import com.braintreepayments.api.ExperimentalBetaApi
-import com.braintreepayments.api.core.PostalAddress
+import com.braintreepayments.api.BrowserSwitchOptions
+import com.braintreepayments.api.core.ExperimentalBetaApi
 
 @ExperimentalBetaApi
 sealed class PayPalVaultEditResult {
```

Comment:
We should still have a `PayPalVaultEditResult` class that is the final result the merchant receives after returning from the browser switch and calling `PayPalClient.edit` - the implementation here should be moved to a separate `PayPalVaultEditAuthRequest` object